src/v/raft/recovery_stm.cc

```diff
@@ -491,7 +491,8 @@ recovery_stm::take_on_demand_snapshot(model::offset last_included_offset) {
       .last_included_term = *term,
       .latest_configuration = std::move(*cfg),
       .log_start_delta = offset_translator_delta(
-        _ptr->_offset_translator.state()->delta(last_included_offset)),
+        _ptr->_offset_translator.state()->delta(
```

Comment:
what a find 🙏

Reply:
Has this translation bug resulted in wrong kafka offsets for the log offsets and that tripped kgo verifier into thinking this is a possible idempotency bug?

Reply:
exactly